Dockerfile

```diff
@@ -6,6 +6,11 @@ COPY ./tsconfig.json /docs/tsconfig.json
 WORKDIR /docs/packages/docs
 RUN npm i
 RUN npm run build
+RUN npm i puppeteer
+
+# install puppeteer/chromium dependencies
+RUN dnf install -y bzip2 fontconfig nss.x86_64 pango.x86_64 libXcomposite.x86_64 libXcursor.x86_64 libXdamage.x86_64 libXext.x86_64 libXi.x86_64 libXtst.x86_64 cups-libs.x86_64 libXScrnSaver.x86_64 libXrandr.x86_64 GConf2.x86_64 alsa-lib.x86_64 atk.x86_64 gtk3.x86_64 libdrm libgbm libxshmfence
+RUN dnf install -y GConf2 nss libXScrnSaver alsa-lib wget
```

Comment:
Not sure it was in the docs.

Reply:
BEcause each `RUN` creates a new docker layer, if we don't expect the layer to change that often it is good to merge certain RUN commands together (to sorta cache them)